### PR TITLE
Tracking within SVCCA analysis for both training and testing

### PR DESCRIPTION
**Pull request by gwaygenomics**
Adding options to visualize training and testing SVCCA results

### CODE REVIEW

File: 4.analyze-components/scripts/nbconverted/5.visualize-svcca.r [hunk 1]
```diff
@@ -9,8 +9,11 @@ svcca_df <- readr::read_tsv(svcca_file,
                             col_types = readr::cols(
                                 .default = readr::col_character(),
                                 svcca_mean_similarity = readr::col_double()))
+print(dim(svcca_df))
 head(svcca_df, 2)
```
Comment (ajlee21) [inline 4.analyze-components/scripts/nbconverted/5.visualize-svcca.r]: if you're printing the dimensions I would add a string of what's being printed

File: 4.analyze-components/scripts/nbconverted/5.visualize-svcca.r [hunk 2]
```diff
@@ -31,52 +34,117 @@ svcca_df$algorithm_2 <- factor(
 
 options(repr.plot.width = 15, repr.plot.height = 9)
 
-for (dataset in c("TARGET", "TCGA", "GTEX")) {
+for(dataset in c("TARGET", "TCGA", "GTEX")) {
```
Comment (ajlee21) [inline 4.analyze-components/scripts/nbconverted/5.visualize-svcca.r]: looks like you have 3 for loops here:) Not sure how long this takes to run, curious if you would want to always run all three datasets everytime?

File: 4.analyze-components/scripts/nbconverted/5.visualize-svcca.r (hunk 1 above)
Comment (gwaygenomics) [reply 4.analyze-components/scripts/nbconverted/5.visualize-svcca.r]: this is being run in a jupyter notebook, so the print statement should be apparent to someone viewing the notebook

File: 4.analyze-components/scripts/nbconverted/5.visualize-svcca.r (hunk 2 above)
Comment (gwaygenomics) [reply 4.analyze-components/scripts/nbconverted/5.visualize-svcca.r]: yeah, I am just plotting and I will want to do it for each dataset each time

**Pull request merged**